src/main/kotlin/agallipo/BoardingCardSorter.kt

```diff
@@ -1,114 +1,114 @@
-package agallipo
+    package agallipo
 
-import java.io.ObjectInputFilter.merge
+    import java.io.ObjectInputFilter.merge
+    //created by alex :)
+    interface BoardingCardSorter {
 
-interface BoardingCardSorter {
+        //This function divides  all the boarding cards into lists of flight-paths, iterating over all the boarding cards,
+        //it first checks if the boarding card can be joined to any journey, if not, it creates a new element in the
+        //list of flighPaths. Then it looks over if any flight path can be joined
 
-    //This function divides  all the boarding cards into lists of flight-paths, iterating over all the boarding cards,
-    //it first checks if the boarding card can be joined to any journey, if not, it creates a new element in the
-    //list of flighPaths. Then it looks over if any flight path can be joined
-
-    fun sortTickets(boardingCards: MutableList<BoardingCard?>): MutableList<MutableList<BoardingCard?>> {
-        var flightPaths = mutableListOf<MutableList<BoardingCard?>>()
-        for (boardingCard in boardingCards) {
-            if (!matchFlightPath(boardingCard, flightPaths))
-                flightPaths.add(mutableListOf<BoardingCard?>(boardingCard))
-            checkJoinFlightPath(flightPaths)
+        fun sortTickets(boardingCards: MutableList<BoardingCard?>): MutableList<MutableList<BoardingCard?>> {
```

Comment:
What if instead of having a MutableList of MutableList of BoardingCards, think on a data type (either created by you or using another data structure) that ease the management of the result?